src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/CommPkgTests.cs

```diff
@@ -12,19 +12,21 @@ public class CommPkgTests
         private CommPkg _dut;
         private const string TestPlant = "PlantA";
         private const string ProjectName = "ProjectName";
+        private static readonly Guid ProjectGuid = new Guid("11111111-2222-2222-2222-333333333341");
         private const int ProjectId = 320;
         private const string CommPkgNo = "Comm1";
         private const string Description = "D1";
         private const string Status = "OK";
         private const string System = "1|2";
-        private readonly Project _project = new(TestPlant, ProjectName, $"Description of {ProjectName}");
+        private readonly Guid CommPkgGuid = Guid.Empty;
```

Comment:
make CommPkgGuid other than empty and Assert it is set in Constructor_ShouldSetProperties()